### ISSUE TITLE
Compatibility with SOAP v2

### ISSUE THREAD

**Issue by kevinwu** [magja/magja#82]
Is this library compatible with SOAP v2?

**Comment by zambrovski** [comment]
No. Current version of Magja is using v1 API. Is there any need on v2?

**Comment by MarcusWolschon** [comment]
Of cause there is.

**Comment by janrkrejci** [comment]
I need this too!  It would be great since I'll be starting a project in a few weeks and I thinks to use this API.
